Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "StaticArrays"
 uuid = "90137ffa-7385-5640-81b9-e52037218182"
-version = "1.5.21"
+version = "1.5.22"
```

Comment:
1.5.22 is already released so it needs to be bumped to 1.5.23 now :slightly_smiling_face: . I think you need to either merge master into your branch or rebase this branch to get a clean merge.

Reply:
*shakes fist* haha will do

Reply:
Done!

Reply:
Thanks!